.github/workflows/lint.yaml

```diff
@@ -17,8 +17,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: "20.x"
```

Comment:
Needed to bump this so that I can use the most recent versions of `chai`. I was running into some ESModule/Common `tsconfig.json` issues without it.